public/js/profileList.js

```diff
@@ -0,0 +1,80 @@
+const openEditProfileForm = (profile) => {
+  // eslint-disable-next-line no-underscore-dangle
+  window.location.href = `${window.location.origin}/views/profileForm.html?profileId=${profile._id}`;
+};
+
+const openNewProfileForm = () => {
+  // eslint-disable-next-line no-underscore-dangle
+  window.location.href = `${window.location.origin}/views/profileForm.html`;
+};
+
+window.onload = () => {
+  const navButton = document.getElementById('profilesNav');
+  const tableContent = document.getElementById('table-content');
+  const addProfileButton = document.getElementById('addProfile');
+
+  // MODAL SETTINGS
+  const modalSection = document.getElementById('modal-section');
```

Comment:
You can move the basic functions of the modal to a separate js file, to be able to share it in the different html without having to repeat the same logic in each js file.

Reply:
modal.js was created, fixed it